src/material-experimental/mdc-slide-toggle/slide-toggle.scss

```diff
@@ -23,6 +23,12 @@
     pointer-events: none;
   }
 
+  // The thumb underlay needs additional specificity to override the focus indicator's relative
+  // positioning.
+  .mdc-switch__thumb-underlay {
+    position: absolute;
```

Comment:
I'm operating under the assumption that we can't rely on stylesheet ordering for CSS specificity, so I included this style so that its specificity (resolves to 0,2,0) beats the global `mat-mdc-strong-focus-indicator` style specificity of 0,1,0.

Reply:
Spoke offline, we can rely upon stylesheet ordering, so removed this change.